BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -341,6 +368,27 @@ Rationale: SNI supports deployment of multiple TLS-protected virtual servers on
       target domain for a given connection; this information leak will be plugged by 
       use of TLS Encrypted Client Hello.
 
+In order to prevent the attacks described in {{ALPACA}}, a receiver who does
+not recognize the presented server name SHOULD NOT continue the handshake and
+instead fail with a fatal-level `unrecognized_name(112)` alert.  Note that this
+recommendation updates Section 3 of {{!RFC6066}}: "If the server understood the
+ClientHello extension but does not recognize the server name, the server SHOULD
+take one of two actions: either abort the handshake by sending a fatal-level
+`unrecognized_name(112)` alert or continue the handshake."
+
```

Comment:
What about the client-side recommendation on the Alpaca web site:
> We also recommend that clients abort the handshake if the server acknowledges the SNI hostname with a different hostname than the one sent by the client. 

Also, maybe we should use "server" instead of "receiver" above. Or if you meant both, please clarify.

Reply:
you are right!  fixed in f9a03d4